rotkehlchen/db/history_events.py

```diff
@@ -401,35 +402,40 @@ def get_history_events(
         base_suffix = f'{HISTORY_BASE_ENTRY_FIELDS}, {EVM_EVENT_FIELDS}, {ETH_STAKING_EVENT_FIELDS} {ALL_EVENTS_DATA_JOIN}'  # noqa: E501
         bindings = []
         type_idx = 1 if group_by_event_ids is True else 0
-        special_free_query = False
-        free_query_group_by = maybe_filter_ignore_asset(filter_query)
+        final_clause = ''
```

Comment:
this is only for grouping and ordering right? rather than final_clause can you come up with a name that is more descreptive? Or at least having a comment explaining what you use this for